adoc/MAIN.release-notes.adoc

```diff
@@ -82,6 +82,20 @@ Update helm on your management workstation as you would do with any other packag
                                                                                 
 Refer to: link:https://documentation.suse.com/sles/15-SP1/single-html/SLES-admin/#sec-zypper-softup[{sls} 15 SP1 Admin Guide: Updating Software with Zypper]
 
+=== Bugs Fixed in 4.1.0 since 4.0.3
+
+* link:https://bugzilla.suse.com/show_bug.cgi?id=1144065[cri-o] - (CVE-2019-10214) VUL-0: CVE-2019-10214: libcontainers-common: library does not enforce TLS connections
```

Comment:
Please change the link formatting to:

`link:<BUGZILLA URL>[bsc#<BUGZILLA ID> - <COMPONENT>]`
e.g.
`link:https://bugzilla.suse.com/show_bug.cgi?id=1144065[bsc#1144065 - cri-o]`

The default link text would be `cri-o` which is not very helpful. If a customer is looking for a specific bugzilla ID that affects them, they can't see those right now without manually checking every link.

Also please provide links to the respective CVE pages on suse.com: https://www.suse.com/security/cve

Reply:
I missed the bug# part.  Have make change to consist with previous bug change and added cve.